packages/cli/src/commands/server/launchDebugger.js

```diff
@@ -69,24 +47,20 @@ function getChromeAppName(): string {
 }
 
 function launchChrome(url: string) {
-  open(url, {app: [getChromeAppName()]}, err => {
-    if (err) {
-      logger.error('Google Chrome exited with error:', err);
-    }
-  });
+  return open(url, {app: [getChromeAppName()], wait: true});
```

Comment:
Test it, It's not working properly, still getting `For a better debugging experience please install Google Chrome from` and I have Chrome installed and my default browser is `Brave`.

I don't think we need to use `wait: true` here, from `open` documentation:
```
wait
Type: boolean

Wait for the opened app to exit before fulfilling the promise. If false it's fulfilled immediately when opening the app.
```

You need to use async/await with open, something like this:

```
async function launchChrome(url: string) {
  const {err} = await open(url, {app: getChromeAppName()});
  if (err) {
    logger.error('Google Chrome exited with error:', err);
  }
}
```

Reply:
Without `wait: true`, it will not work as we need it to: no errors will be reported to the calling code and try/catch will not make any difference. Here is an extract from `open`'s code:

```
	const subprocess = childProcess.spawn(command, cliArguments, childProcessOptions);

	if (options.wait) {
		return new Promise((resolve, reject) => {
			subprocess.once('error', reject);

			subprocess.once('close', exitCode => {
				if (exitCode > 0) {
					reject(new Error(`Exited with code ${exitCode}`));
					return;
				}

				resolve(subprocess);
			});
		});
	}

	subprocess.unref();

	return subprocess;
```

It seems to be working well with my modifications for me . :)

Reply:
Ah sorry my bad, works great!